packages/react-core/src/components/SearchInput/SearchInput.tsx

```diff
@@ -277,11 +277,9 @@ const SearchInputBase: React.FunctionComponent<SearchInputProps> = ({
 
   if (!!onSearch || attributes.length > 0 || !!onToggleAdvancedSearch) {
     if (attributes.length > 0) {
-      return (
-        <div className={css(className, styles.searchInput)} ref={searchInputRef} {...props}>
-          {buildSearchTextInputGroupWithExtraButtons()}
+      const advancedSearch = (
+        <span>
```

Comment:
we need a native HTML markup here, otherwise, the popper is not going to calculate the position and set the item absolute